libs/clients/ums-cost-of-living-calculator/src/lib/fetch/apis/CostOfLivingCalculatorApi.ts

```diff
@@ -0,0 +1,42 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { UmbodsmadurSkuldaraClientConfig } from '../../umsCostOfLivingCalculator.config'
+import { ConfigType } from '@nestjs/config'
+export interface CostOfLivingCalculatorRequest {
+  fjMerking: string
+  fot: number
+  laek: number
+  matur: number
+  onnurthon: number
```

Comment:
Just making sure that there's not typo here.

Is it definitely `onnurthon` and not `onnurthjon`?